### PR TITLE
Final

### PR DESCRIPTION
**Pull request by jsuvorovs**
#2

### CODE REVIEW

File: 08_04_2020_WordTransformer/src/Rule5.java [hunk 1]
```diff
@@ -1,4 +1,5 @@
 public class Rule5 extends Rule3 implements Transform {
+    //Rule 5
     @Override
     public boolean check(String split) {
         if (split.length() == 5) return true;
```
Comment (sergeylukichev) [inline 08_04_2020_WordTransformer/src/Rule5.java:5]: `return split.length() == 5;`

File: 08_04_2020_WordTransformer/src/Rule3.java [hunk 2]
```diff
@@ -1,4 +1,5 @@
 public class Rule3 implements Transform {
+    //Rule 3
     @Override
     public boolean check(String split) {
         if (split.length() == 3) return true;
```
Comment (sergeylukichev) [inline 08_04_2020_WordTransformer/src/Rule3.java:5]: `return split.length() == 3;`

File: 08_04_2020_WordTransformer/src/Transform.java [hunk 3]
```diff
@@ -1,4 +1,5 @@
 public interface Transform {
+    //Interface
```
Comment (sergeylukichev) [inline 08_04_2020_WordTransformer/src/Transform.java:2]: this word is not needed.

**Comment by sergeylukichev** [comment]
Merge is not possible because there are conflicting files. You need to resolve conflicts first, then merge.

File: 08_04_2020_WordTransformer/src/Transform.java (hunk 3 above)
Comment (jsuvorovs) [reply 08_04_2020_WordTransformer/src/Transform.java:2]: I did it so I could make a new commit for each file. I know that we must not do this.

**Pull request merged**